### DISCUSSION TITLE
Headless Chakra ?

### DISCUSSION THREAD

**Discussion by VaZark** [chakra-ui/chakra-ui#6914] [Ideas] [answered]
Have you given thought to separating out the lib into Headless Chakra UI components and Design Language library? 

While styling and functionality are part of the overall UX of the system, it would amazing if we could use Chakra's amazing components with custom design languages like TailwindCSS. (I primarily use it for layouting)

This would also help with the enormous bundle size with all the variants generated when building a project with chakra as a dependency.

**Comment by atharva-upadhye** [comment, marked as answer]
We now have it. Check [Ark UI](https://ark-ui.com/)

**Reply by VaZark** [reply]
That looks awesome! Just a quick question.. is it CSS-driven or  CSS-in-JS?
